client/dune

```diff
@@ -4,3 +4,10 @@
   (modules     dns_client)
   (libraries   dns.cache domain-name dns randomconv rresult)
   (wrapped     false))
+
+(library
+  (name        dns_client_tls)
+  (public_name dns-client.tls)
+  (modules     dns_client_tls)
+  (libraries   domain-name mirage-flow dns dns-client randomconv rresult tls)
+  (wrapped     false))
```

Comment:
```suggestion

```

Reply:
maybe this whole thing should just be commented out